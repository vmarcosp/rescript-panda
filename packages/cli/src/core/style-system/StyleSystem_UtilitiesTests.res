open Vitest
open StyleSystem_Utilities

describe("StyleSystem_Utilities", () => {
  test(".rgb(...)", _ => {
    open Expect

    expect(rgb(255, 255, 255))->toBe("[rgb(255, 255, 255)]")
  })

  test(".rgba(...)", _ => {
    open Expect

    expect(rgba(255, 255, 255, 255))->toBe("[rgba(255, 255, 255, 255)]")
  })
})
