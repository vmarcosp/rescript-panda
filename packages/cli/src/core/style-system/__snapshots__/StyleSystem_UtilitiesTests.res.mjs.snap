// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`StyleSystem_Utilities > .hsl(...) 1`] = `"[hsl(255, 255, 255)]"`;

exports[`StyleSystem_Utilities > .hsla(...) 1`] = `"[hsla(255, 255, 255, 255)]"`;

exports[`StyleSystem_Utilities > .rgb(...) 1`] = `"[rgb(255, 255, 255)]"`;

exports[`StyleSystem_Utilities > .rgba(...) 1`] = `"[rgba(255, 255, 255, 255)]"`;
